dashboard/cron.py

```diff
@@ -480,14 +485,30 @@ def submission(self):
 
         # loop through multiple course ids
         # filter out not released grades (submission_dim.posted_at date is not null) and partial grades (submission_dim.workflow_state != 'graded')
-        status += util_function(queries['submission'],
-                                'submission',
-                                {
-                                    'course_ids': tuple(self.valid_locked_course_ids),
-                                    'canvas_data_id_increment': settings.CANVAS_DATA_ID_INCREMENT,
-                                    'time_zone': settings.TIME_ZONE
-                                })
+        query_params = {
+                        'course_ids': self.valid_locked_course_ids,
+                         'time_zone': settings.TIME_ZONE,
+                        'canvas_data_id_increment': settings.CANVAS_DATA_ID_INCREMENT,
+                        }
+        Session = sessionmaker(bind=data_warehouse_engine)
+        try:
+    # Create a session
+            with Session() as session:
+                # Execute the first query to create the temporary table
+                session.execute(text(queries['submission']).bindparams(**query_params))
+
+                # Execute the second query using the temporary table
+                result = session.execute(text(queries['submission_with_avg_score']))
+                df = pd.DataFrame(result.fetchall(), columns=result.keys())
+                df = df.drop_duplicates(keep='first')
+                df.to_sql(con=engine, name='submission', if_exists='append', index=False)
+
```

Comment:
Split up the Submission into 2 since new postgres driver did not supported 2 queries in one. I have details on this in the PR description